### PR TITLE
Change 'max' and 'guaranteed' memory when editing VM's memory

### PR DESCRIPTION
**Pull request by mareklibra**
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/175

### CODE REVIEW

**Comment by mareklibra** [comment]
@jelkosz , can you please have a look at the computation?

Recently user can't modify max and guaranteed memory in web-ui, it's a planed feature.
From user perspective, this seems to be advanced VM configuration and we need to be careful when/how we add it into the web-ui. So not within this PR.

File: src/components/VmDialog/index.js [hunk 1]
```diff
@@ -128,6 +131,21 @@ class VmDialog extends React.Component {
     return last ? last.message : undefined
   }
 
+  getMemoryPolicy () {
+    const cluster = this.getCluster()
+    console.log('getMemoryPolocy, overCommit: ', cluster.getIn(['memoryPolicy', 'overCommitPercent']))
```
Comment (jniederm) [inline src/components/VmDialog/index.js]: Is this intentional? Do we want such log?

File: src/components/VmDialog/index.js [hunk 2]
```diff
@@ -128,6 +131,21 @@ class VmDialog extends React.Component {
...
+    let guaranteed = (cluster && cluster.getIn(['memoryPolicy', 'overCommitPercent']))
+      ? (this.state.memory * (100 / cluster.getIn(['memoryPolicy', 'overCommitPercent']))) : this.state.memory
+
+    const memoryPolicy = {
+      'max': this.state.memory * MAX_VM_MEMORY_FACTOR,
+      'guaranteed': Math.round(guaranteed),
+    }
+    logDebug('getMemoryPolicy() resulting memory_policy: ', memoryPolicy)
```
Comment (jniederm) [inline src/components/VmDialog/index.js]: similar

File: src/components/VmDialog/index.js (hunk 1 above)
Comment (mareklibra) [inline src/components/VmDialog/index.js]: Thanks, leftover

File: src/components/VmDialog/index.js (hunk 2 above)
Comment (mareklibra) [inline src/components/VmDialog/index.js]: This was intentional for DEBUG level

File: src/components/VmDialog/index.js [hunk 3]
```diff
@@ -128,6 +131,20 @@ class VmDialog extends React.Component {
     return last ? last.message : undefined
   }
 
+  getMemoryPolicy () {
+    const cluster = this.getCluster()
+    const overCommitPercent = cluster && cluster.getIn(['memoryPolicy', 'overCommitPercent'])
+    let guaranteed = overCommitPercent ? (this.state.memory * (100 / overCommitPercent)) : this.state.memory
+
+    const memoryPolicy = {
+      'max': this.state.memory * MAX_VM_MEMORY_FACTOR,
```
Comment (jelkosz) [inline src/components/VmDialog/index.js]: is this only for next run or also hotplug? If it is for next_run only than it is ok, if it is about hotplug than the "max" should stay as is.

File: src/components/VmDialog/index.js [hunk 4]
```diff
@@ -128,6 +131,20 @@ class VmDialog extends React.Component {
     return last ? last.message : undefined
   }
 
+  getMemoryPolicy () {
+    const cluster = this.getCluster()
+    const overCommitPercent = cluster && cluster.getIn(['memoryPolicy', 'overCommitPercent'])
+    let guaranteed = overCommitPercent ? (this.state.memory * (100 / overCommitPercent)) : this.state.memory
```
Comment (jelkosz) [inline src/components/VmDialog/index.js]: this is ok.

File: src/components/VmDialog/index.js (hunk 3 above)
Comment (mareklibra) [inline src/components/VmDialog/index.js]: Recently it's for next_run only

**Pull request merged**